CHANGELOG-2.1.x.md

```diff
@@ -1,6 +1,9 @@
 # Within 2.1
 
 ## 2.1.0
+ - BC-Break: `quantity` Field is now a `text` field instead of `number`:
```

Comment:
what quantity field?

Reply:
The frontend quantity field in the addToCartForm and within the cartForm

Reply:
why not a number that allows precision?

Reply:
https://caniuse.com/#feat=input-number

Mobile:
- UI widget does not include increment/decrement buttons.
- UI widget does not take the "step", "min" or "max" attributes into account.

Reply:
changed back to number field